Refactor : 스케줄러 리펙토링

parkjihwanKr opened the pull request:
- [x]  QueryDSL을 class별 관리
- [x]  domainRepository를 사용하지 않고 Service의 method를 사용하여 도메인 별 repository에 접근하기
- [x] 해당 Service, Facade 주석 처리로 설명 추가
- [x] 해당 삭제에 대하여 오전 1시에 deleted_at이 1년 지났거나 reservation같은 경우는 cancel_at 또는 check_out 시간이 1년 지날 경우에 삭제

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

Scheduler의 도메인 Repository 의존성 주입 제거, UsersServiceImplV1에 deleteUsers 메서드 구현, Scheduler 구현 후, 제거 예정

Fixed : #104

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

conflict로 인한 merge

Fixed : #104

parkjihwanKr committed:
Refactor : [Deletion_Management] Scheduler 리팩토링

 QueryDSL을 class별 관리,
 DomainRepository를 사용하지 않고 Service의 method를 사용하여 도메인 별 repository에 접근하기

Fixed : #104

parkjihwanKr committed:
Refactor : [Deletion_Management] Scheduler 리팩토링

Scheduler 설정 변경

Fixed : #152

parkjihwanKr committed:
Merge branch 'develop' into feat/deletion_management

parkjihwanKr committed:
Refactor : [Deletion_Management] Scheduler 리팩토링

Scheduler Conflict 해결

Fixed : #152

parkjihwanKr committed:
Merge remote-tracking branch 'origin/feat/deletion_management' into feat/deletion_management

# Conflicts:
#	src/main/java/com/example/airdns/domain/payment/repository/PaymentRepository.java
#	src/main/java/com/example/airdns/domain/reservation/service/ReservationService.java
#	src/main/java/com/example/airdns/domain/reservation/servicefacade/ReservationServiceFacade.java
#	src/main/java/com/example/airdns/domain/reservation/servicefacade/ReservationServiceFacadeImplV1.java
#	src/main/java/com/example/airdns/domain/user/service/UsersService.java

parkjihwanKr committed:
Merge branch 'develop' into feat/deletion_management

parkjihwanKr committed:
Refactor : [Deletion_Management] Scheduler 리팩토링

Scheduler Conflict 해결

Fixed : #152

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

Scheduler Conflict 해결

Fixed : #152

parkjihwanKr committed:
Merge remote-tracking branch 'origin/feat/deletion_management' into feat/deletion_management

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

Scheduler Conflict 해결

Fixed : #152

parkjihwanKr committed:
Merge branch 'develop' of https://github.com/AirDnS/airDnS into feat/deletion_management

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

해당 Service, Facade 주석 처리

Fixed : #152

parkjihwanKr committed:
Merge branch 'develop' into feat/deletion_management

Kim-Jong-Gyu commented on src/main/java/com/example/airdns/domain/deleteinfo/entity/DeleteReservationsInfo.java:
@Column 어노테이션을 주석을 하신 이유가 있을까요???

parkjihwanKr replied:
해당 사항은 지연 로딩에 의해서 room이 user의 정보를 못 받아와서 임시로 처리한 내용입니다! 이를 수정해서 다시 올리도록 하겠습니다.

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

Conflict 해결

Fixed : #152

parkjihwanKr committed:
Merge remote-tracking branch 'origin/feat/deletion_management' into feat/deletion_management

# Conflicts:
#	src/main/java/com/example/airdns/domain/user/service/UsersService.java

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

Conflict 해결

Fixed : #152

parkjihwanKr committed:
Fix : [Deletion_Management] Scheduler 리팩토링

각 Domain별 필요한 정보 추가하여 저장
Room -> ownerName
Reservation -> roomName, reserverName
Payment -> paymentKey

Resolved to : #152
